src/components/PackageCard/Details.tsx

```diff
@@ -30,12 +36,26 @@ export const Details: FunctionComponent = () => {
     author,
     metadata: { date },
     version,
+    name,
   } = usePackageCard();
 
+  const { data } = useStats();
+  const downloads: number | undefined = useMemo(() => {
+    return data?.packages?.[name]?.downloads?.npm;
+  }, [name, data]);
```

Comment:
Nitpick, but I think you can drop the useMemo since we're only doing property access for a number value here. A reason we would potentially want to memoize this value is if we were accessing an object (or array) and needed to prevent it from causing unwanted renders due to react's shallow object equality checks i.e `object1 === object2`

Reply:
Makes sense! 👍